java/src/org/openqa/selenium/ie/InternetExplorerDriver.java

```diff
@@ -130,25 +135,30 @@ public InternetExplorerDriver(InternetExplorerDriverService service) {
    * @param options The options required from InternetExplorerDriver.
    */
   public InternetExplorerDriver(InternetExplorerDriverService service,
-                                InternetExplorerOptions options) {
+                                InternetExplorerOptions options,
+                                ClientConfig clientConfig) {
     if (options == null) {
       options = new InternetExplorerOptions();
     }
     if (service == null) {
       service = InternetExplorerDriverService.createDefaultService();
     }
-    run(service, options);
+    if (clientConfig == null){
+      clientConfig = ClientConfig.defaultConfig();
+    }
+
+    run(service, options, clientConfig);
   }
 
   @Beta
   public static RemoteWebDriverBuilder builder() {
     return RemoteWebDriver.builder().oneOf(new InternetExplorerOptions());
   }
 
-  private void run(InternetExplorerDriverService service, Capabilities capabilities) {
+  private void run(InternetExplorerDriverService service, Capabilities capabilities, ClientConfig clientConfig) {
     assertOnWindows();
 
-    setCommandExecutor(new DriverCommandExecutor(service));
+    setCommandExecutor(new DriverCommandExecutor(service, capabilities, clientConfig));
```

Comment:
I see compilation errors here because we are invoking a protected constructor which is visible only to child classes. Please fix this.

Reply:
We should have added a new 2 arg constructor in `DriverCommandExecutor` that accepts a `DriverService` and a `ClientConfig` object which should be called from here.